src/Mvc/Mvc.ApiExplorer/src/DependencyInjection/EndpointMethodInfoApiExplorerServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.AspNetCore.Mvc.Infrastructure;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Extensions for configuring ApiExplorer using <see cref="Endpoint.Metadata"/>.
+    /// </summary>
+    public static class EndpointMetadataApiExplorerServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Configures ApiExplorer using <see cref="Endpoint.Metadata"/>.
+        /// </summary>
+        /// <param name="services">The <see cref="IServiceCollection"/>.</param>
+        public static void AddEndpointsApiExplorer(this IServiceCollection services)
```

Comment:
Could we do a super quick email review for the name? See if others have a more compelling name suggestion? This is alright, but doesn't feel great.

Reply:
```suggestion
        public static IServiceCollection AddEndpointsApiExplorer(this IServiceCollection services)
```

It matches other Add* APIs

Reply:
I was planning to discuss this as part of our normal Monday API review meeting, but I'll also send an email.

Reply:
How about `AddMinimalActionEndpointsApiExplorer`? like the type that `Map` method returns `MinimalActionEndpointConventionBuilder`.

https://github.com/dotnet/aspnetcore/blob/ce3a74883c7276e8747200ca8f06bba8b7211427/src/Http/Routing/src/Builder/MinimalActionEndpointRouteBuilderExtensions.cs#L139